formidling/src/main/java/no/nav/ung/sak/formidling/template/dto/innvilgelse/beregning/BarnetilleggDto.java

```diff
@@ -0,0 +1,8 @@
+package no.nav.ung.sak.formidling.template.dto.innvilgelse.beregning;
+
+public record BarnetilleggDto(
+    int antall,
+    int dagsatsTillegg,
+    long dagsatsMedTillegg
```

Comment:
Hva er forskjellen mellom `dagsatsTillegg` og `dagsatsMedTillegg`?

Reply:
Never mind, så hva de var når jeg leste template filen. 
Var litt forvirrende med to nesten like variabelnavn.

Reply:
kan evt. endre dagsatsMedTillegg til dagsatsInklBarnetillegg

Reply:
eller endre dagsatsTillegg til dagsats siden det er i kontekst av `BarneTilleggDto` og dagsatsMedTillegg til dagsatsTotal